test/queries/getInterests.test.js

```diff
@@ -0,0 +1,16 @@
+const test = require('tape');
+const { interests: { getInterests } } = require('../../server/database/queries');
+const dbBuild = require('../../server/database/config/dbbuild');
+
+test('Testing for the interests query', (t) => {
+  dbBuild()
+    .then(() => getInterests())
+    .then((res) => {
+      t.equal(JSON.stringify(res.rows).includes('name'), true, 'There is a name if we have interests');
```

Comment:
Why JSON.strinfigy()! Make a serious test or don't make any tests at all :grimacing: Test, for example, the length of interests array, ot their values.

Reply:
We don't have a specific number of interests yet so I just made it into a string so I can check if it includes something that I would expect to see in the response

Reply:
thx can review the tests for all PR  @IsraaSulaiman  if you have time for sure